### PR TITLE
Prevent glGetError infinity

### PR DESCRIPTION
**Pull request by metarutaiga**
if EGL context is not available, the glGetError always return GL_INVALID_OPERATION.

I use my simple EGL to WGL wrapper, and found an infinite loop when destroying interpreter.
https://github.com/metarutaiga/TensorFlowLiteReduce/blob/master/third_party/EGL/egl_windows.c

### CODE REVIEW

**Comment by metarutaiga** [comment]
@googlebot I signed it!

**Comment by gbaned** [comment]
@impjdi Can you please review this PR ? Thanks!

File: tensorflow/lite/delegates/gpu/gl/gl_errors.cc [hunk 1]
```diff
@@ -67,6 +67,10 @@ absl::Status GetOpenGlErrors() {
   if (error2 == GL_NO_ERROR) {
     return absl::InternalError(ErrorToString(error));
   }
+  EGLContext context = eglGetCurrentContext();
```
Comment (impjdi) [inline tensorflow/lite/delegates/gpu/gl/gl_errors.cc:70]: Is this the right location for checking?  shouldn't this check be at L.62, before the first glGetError?

Comment (metarutaiga) [reply tensorflow/lite/delegates/gpu/gl/gl_errors.cc:70]: https://www.khronos.org/opengl/wiki/OpenGL_Error
Note: No OpenGL function call is valid when an OpenGL Context is not made current. Depending on the platform, a call to glGetError without having a context made current may crash or return any value, including indefinitely returning a valid OpenGL error code. So if you see an infinite loop in such a function, this is likely the reason why.

Hmmm, I move it upper.

**Review by impjdi — APPROVED** [review]

**Comment by impjdi** [comment]
FYI this PR broke internal Google apps and will be rolled back.

**Pull request merged**